test/server/index.js

```diff
@@ -143,11 +109,21 @@ app.get("/ping", async (req, res) => {
 });
 
 app.post("/startst", async (req, res) => {
+    if (req.body && req.body.configUpdates) {
+        for (const update of req.body.configUpdates) {
+            await setKeyValueInConfig(update.key, update.value);
+        }
+    }
     let pid = await startST();
     res.send(pid + "");
 });
 
 app.post("/beforeeach", async (req, res) => {
```

Comment:
When testing with a backend SDK, there are two servers:
- `TEST_SERVER_BASE_URL`: This is a node server in the auth-react package. This is in charge of starting / stopping ST etc...
- `TEST_APPLICATION_SERVER_BASE_URL`: This is the node server in the supertokens-node package. Or a go server in supertokens-golang package. This is where the for-tests app will query. Therefore, these operations below will need to be carried out in this server while testing. This in turn implies that the frontend will have to query the `/beforeeach` route of the `TEST_APPLICATION_SERVER_BASE_URL` as well - so this will require a change in the `before(async function () {` block of the frontend test files.